### DISCUSSION TITLE
Multiple-instrument Callbacks

### DISCUSSION THREAD

**Discussion by gabswb** [open-telemetry/opentelemetry-cpp#1751] [Q&A]
Hi, 
Does opentelemetry-cpp support multiple-instrument callbacks as mentionned in the specification ?  
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#multiple-instrument-callbacks 
I'm asking because I didn't found it by myself.
Thanks

**Comment by lalitb** [comment]
@gabswb  It was left out of the Metrics GA release as was tagged optional by specs. Have now created the issue #1753 for this. Shouldn't be a bigger change, but need to prioritize for one of the upcoming releases.
